### PR TITLE
fix format error

### PR DESCRIPTION
**Pull request by xieyanker**
fix format error as follows:

image

image

### CODE REVIEW

**Comment by xieyanker** [comment]
https://deploy-preview-19925--kubernetes-io-master-staging.netlify.com/zh/docs/tasks/configure-pod-container/static-pod/
&&
https://deploy-preview-19925--kubernetes-io-master-staging.netlify.com/zh/docs/tutorials/configuration/configure-redis-using-configmap/

/assign @tengqm

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -49,14 +49,6 @@ kubelet 会尝试通过 Kubernetes API 服务器为每个静态 Pod 自动创建
...
 
-<!--
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
-This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,
-and that your nodes are running the Fedora operating system.
-Instructions for other distributions or Kubernetes installations may vary.
--->
-
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/static-pod.md]: The correct way of fixing this is to move `<!--` after the `include` shortcode.

Comment (xichengliudui) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: great

Comment (xichengliudui) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}	
\

Comment (xieyanker) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: @tengqm @xichengliudui Thanks for your correction! I have repushed it.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**